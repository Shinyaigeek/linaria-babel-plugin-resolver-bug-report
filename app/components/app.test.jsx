import React from "react";
import { test, expect } from "@jest/globals";
import { create } from "react-test-renderer";
import App from "./app";

test("take app snapshot", () => {
    expect(create(<App />)).toMatchSnapshot();
})