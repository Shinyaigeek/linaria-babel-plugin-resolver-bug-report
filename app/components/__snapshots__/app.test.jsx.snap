// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`take app snapshot 1`] = `
<div
  className="dobi3i2"
>
  This is error report
</div>
`;
